From 15c751c63b10a2e161d6f2c9f569301893b8c4c1 Mon Sep 17 00:00:00 2001
From: Michael Scott <mike@foundries.io>
Date: Sat, 26 Mar 2022 21:13:21 -0700
Subject: [PATCH] imx8mn-var-som: adjust emmc and sd dev

When testing u-boot, the SD card shows up as "mmc dev 1" and emmc is "mmc dev 2".

Signed-off-by: Michael Scott <mike@foundries.io>
---
 include/configs/imx8mn_var_som.h | 8 ++++----
 1 file changed, 4 insertions(+), 4 deletions(-)

diff --git a/include/configs/imx8mn_var_som.h b/include/configs/imx8mn_var_som.h
index 9c44ded2e0..f950043564 100644
--- a/include/configs/imx8mn_var_som.h
+++ b/include/configs/imx8mn_var_som.h
@@ -76,8 +76,8 @@
 	CONFIG_MFG_ENV_SETTINGS_DEFAULT \
 	"initrd_addr=0x43800000\0" \
 	"initrd_high=0xffffffffffffffff\0" \
-	"emmc_dev=1\0" \
-	"sd_dev=0\0" \
+	"emmc_dev=2\0" \
+	"sd_dev=1\0" \
 
 /* Initial environment variables */
 #define CONFIG_EXTRA_ENV_SETTINGS \
@@ -206,8 +206,8 @@
 	(CONFIG_SYS_INIT_RAM_ADDR + CONFIG_SYS_INIT_SP_OFFSET)
 
 #define CONFIG_ENV_OVERWRITE
-#define CONFIG_SYS_MMC_ENV_DEV		1   /* USDHC2 */
-#define CONFIG_MMCROOT			"/dev/mmcblk1p2"  /* USDHC2 */
+#define CONFIG_SYS_MMC_ENV_DEV		2   /* USDHC2 */
+#define CONFIG_MMCROOT			"/dev/mmcblk2p2"  /* USDHC2 */
 
 /* Size of malloc() pool */
 #define CONFIG_SYS_MALLOC_LEN		((CONFIG_ENV_SIZE + (2*1024) + (16*1024)) * 1024)
-- 
2.35.1

